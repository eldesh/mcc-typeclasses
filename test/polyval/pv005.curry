-- This example shows that non-expansive expressions may involve local
-- let declarations (provided that their bindings are non-expansive
-- themselves).
main = (list False, list 'a')
  where list = \x -> [(n,x)] where n = 1
