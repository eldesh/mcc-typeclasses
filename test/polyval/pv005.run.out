([(1,False)],[(1,'a')])
